### PR TITLE
Add documentation for Alarm Control Panel

### PR DESCRIPTION
**Pull request by avee87**
## Proposed change

Currently this documentation is quite useless, adding info about attributes and services.

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [x] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: 
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: fixes #

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards][].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

File: source/_integrations/alarm_control_panel.markdown [hunk 1]
```diff
@@ -10,3 +10,31 @@ ha_domain: alarm_control_panel
...
+
+- `code_format` - i.e., `text`, `number` or `null` if no code is needed
+- `code_arm_required` - whether code is required to control the alarm
+- `supported_features` - a bitmask representing which features are supported by the alarm. See values for each feature below. For example, if alarm supports "arm home", "arm away" and "trigger" features this value will be 11 (1 + 2 + 8).
+  - 1 - arm home
+  - 2 - arm away
+  - 4 - arm night
+  - 8 - trigger
+  - 16 - custom bypass
+  - 32 - arm vacation
```
Comment (frenck) [inline source/_integrations/alarm_control_panel.markdown]: I think this is a bit too advanced to report. An end-user should not have to worry about internal bitmasks.

File: source/_integrations/alarm_control_panel.markdown [hunk 2]
```diff
@@ -10,3 +10,31 @@ ha_domain: alarm_control_panel
 
 Home Assistant can give you an interface which is similar to a classic alarm system.
 Please see [manual alarm](/integrations/manual) for alarm configuration.
+
+### Attributes
+
+Alarm system exposes the following attributes:
+
+- `code_format` - i.e., `text`, `number` or `null` if no code is needed
```
Comment (frenck) [inline source/_integrations/alarm_control_panel.markdown]: This is a frontend concern, why would we list that here?

File: source/_integrations/alarm_control_panel.markdown (hunk 1 above)
Comment (avee87) [reply source/_integrations/alarm_control_panel.markdown:26]: If this is purely internal, why expose it as an attribute? But yeah, it's probably not useful for 99% of people.

Comment (avee87) [reply source/_integrations/alarm_control_panel.markdown]: If this is internal, why expose it as an attribute? But sure, it's probably not useful for 99% of people...

File: source/_integrations/alarm_control_panel.markdown (hunk 2 above)
Comment (avee87) [reply source/_integrations/alarm_control_panel.markdown]: Mostly for the sake of completeness. Will remove...

File: source/_integrations/alarm_control_panel.markdown (hunk 1 above)
Comment (frenck) [reply source/_integrations/alarm_control_panel.markdown]: > If this is internal, why expose it as an attribute?

The frontend uses these attributes. They are hidden from the frontend towards the end user. It's a transport mechanism.

**Review by frenck — APPROVED** [review]
Thanks, @avee87 👍

**Pull request merged**